### DISCUSSION TITLE
Inference-light suggestions for cloud-based GPU

### DISCUSSION THREAD

**Discussion by eriktaylor** [MIC-DKFZ/nnUNet#992] [Q&A]
Hello,

Great work on this project. nnUnet is working well for my new segmentation use case.

Do you have any recommendations for deploying a light version of nnUnet to open source this trained application specifically for inference?

I've been training on Google colab GPU using a forked repo and wonder if you have any suggestions for a light repo that could be deployed in Google colab or the like for cloud-based inference.

**Comment by FabianIsensee** [comment]
Honestly I don't know. I think nnU-net works fine for inference as well. Yes the default values are compute intensive (maximize segmentation performance) but you can easily tweak them to reduce computation time
